doc/api/process.md

```diff
@@ -212,12 +221,13 @@ behavior. Alternatively, change the [`process.exitCode`][] in the
 provided exit code. Otherwise, in the presence of such handler the process will
 exit with 0.
 
-The listener function is called with the `Error` object passed as the only
-argument.
-
 ```js
-process.on('uncaughtException', (err) => {
-  fs.writeSync(1, `Caught exception: ${err}\n`);
+process.on('uncaughtException', (err, origin) => {
+  fs.writeSync(
+    1,
+    `Caught exception: ${err}\n` +
+    `Exception originated from unhandled rejection: ${origin}`
```

Comment:
Would this variant be appropriate only for boolean type? Maybe something like this?
```suggestion
    `Exception origin: ${origin}`
```